src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -53,6 +54,21 @@ def __init__(
     ):
         super().__init__()
         scheduler = scheduler.set_format("pt")
+
+        if hasattr(scheduler.config, "steps_offset") and scheduler.config.steps_offset != 1:
```

Comment:
@jonatanklosko - Currently the slow tests are broken because we haven't (and can't yet) update the configs online. So for now I think we need to do some deprecation warning here - ok for you?

Reply:
Perfect!

Our of curiosity, is there an issue with updating the configs now? I assume an additional property in the config doesn't hurt even if ignored by previous versions?

Reply:
Most users are on the `0.3.0` version therefore will not make use of this code **but** will nevertheless use the newest config file. Given the high usage of stable diffusion: https://huggingface.co/CompVis/stable-diffusion-v1-4 (>500k downloads per month), I think it would be better to not burden users with an extra config parameter that should throw a warning in `0.3.0` .

So, it would indeed not break anything, but an unused config parameter should throw a warning (haven't checked though whether our code actually works correctly here), which will probably scare users.

Does that make sense?

Reply:
Ah totally, I didn't realise the extra option would cause a warning :)